.github/workflows/build-python.yaml

```diff
@@ -31,6 +31,10 @@ jobs:
 
         run: |
           docker run --rm --privileged torizon/binfmt
+          if ($env:PROJECT_ARCH -eq "arm64") { $env:TORIZON_ARCH = "aarch64" }
+          if ($env:PROJECT_ARCH -eq "armhf") { $env:TORIZON_ARCH = "armhf" }
+          echo "TORIZON_ARCH=$env:TORIZON_ARCH" | `
+            Out-File -FilePath $env:GITHUB_ENV -Encoding utf8 -Append
```

Comment:
is not easier only set the `config:torizon_arch`?

Reply:
I think so, I will try to use it